recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ test:
   commands:
     - test -f "$PREFIX/lib/liboclgrind-rt-icd${SHLIB_EXT}"         # [unix]
     - if not exist %LIBRARY_BIN%\\oclgrind-rt-icd.dll exit 1       # [win]
-    - oclgrind --help
+    - oclgrind --help  # [build_platform == target_platform]
```

Comment:
why? tests are disabled anyways for osx-arm64

Reply:
Where tests are disabled?